terraso_backend/apps/core/models/users.py

```diff
@@ -114,6 +115,36 @@ def soft_delete_policy_action(self, **kwargs):
     def is_group_manager(self, group_id):
         return self.memberships.managers_only().filter(group__pk=group_id).exists()
 
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}".strip()
+
+    def name_and_email(self):
+        return f"'{self.full_name()}' <{self.email}>"
+
+    def notifications_enabled(self):
+        preferences = self.preferences.filter(key="notifications")
+        if len(preferences) != 1:
+            return False
+
+        try:
+            return preferences[0].value.lower() == "true"
+        except AttributeError:
```

Comment:
Question: When is thie expection tirggered?

Reply:
this happens if `.value` fails. It's possible I don't need the try/except block here.

Reply:
Yes, that is what I am thiking I think we should use expections for case that we can't handle in a different way, maybe check if the value exists before can be an option